docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/jira.md

```diff
@@ -0,0 +1,659 @@
+---
+sidebar_label: jira
+description: Integrates with Jira to manage issues, comments, and track promotion workflows.
+---
+
+<span class="tag enterprise"></span>
+<span class="tag beta"></span>
+# `jira`
+
+:::info Enterprise Feature
+This promotion step is only available in Akuity Promote, you can use this feature in Kargo v1.6+ offered on [Akuity Platform](https://akuity.io/akuity-platform).
+:::
+
+The `jira` promotion step provides comprehensive integration with Jira, allowing you to create, update, delete, and search for issues, manage comments, and track promotion workflows. This is particularly useful for maintaining traceability between your promotion processes and project management activities.
+
+This promotion step supports various operations including issue management, comment handling, and status tracking, making it a powerful tool for promotion workflows that require coordination with project management systems.
+
+## Credentials Configuration
+
+All Jira operations require proper authentication credentials stored in a Kubernetes `Secret`.
+
+| Name                     | Type     | Required | Description                                                                                                                  |
+| ------------------------ | -------- | -------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `credentials.secretName` | `string` | Y        | Name of the `Secret` containing the Jira credentials.                                                                        |
+| `credentials.namespace`  | `string` | N        | Namespace containing the credentials `Secret`. This can be the Project's namespace, the cluster secrets namespace (`kargo-cluster-secrets`) or any of the configured global credential namespaces. If empty, the Project's namespace is assumed.|
+
+The referenced `Secret` should contain the following keys:
+
+- `domain`: The domain of your Jira instance or Jira api (e.g., `https://yourcompany.atlassian.net`)
+- `username`: Your Jira username or email
+- `password`: Your Jira API token or password
+
+## Issue Management
```

Comment:
More of a question than a comment. Does Jira use the terms "issue" and "ticket" completely interchangeably? It seems we're using "issue" in some places, but "ticket" in others (mostly in examples).